.github/workflows/cicd.yaml

```diff
@@ -55,6 +56,7 @@ jobs:
         run: |
           if [[ "$libsChanged" == "true" ]]; then
             echo "services=$(ls services | grep -v 'impulse_svc' | uniq | jq -R . | jq -sc .)" >> $GITHUB_OUTPUT
+            echo "libs=$(ls libs | grep -v 'proto_helpers' | uniq | jq -R . | jq -sc .)" >> $GITHUB_OUTPUT
```

Comment:
just add a comment explaining this line, and why proto_helpers should be excluded